docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-modal.md

```diff
@@ -0,0 +1,154 @@
+           modal.Secret.from_name("postgres-read-replica-prod"),
+       ],
+       # run this pipeline daily at 6:24 AM
+       schedule=modal.Cron("24 6 * * *"),
+       timeout=3000,
+   )
+   def load_table_from_database(
+       table: str,
+       incremental_col: str,
+       dev: bool = False,
+   ) -> None:
+       # Placeholder for future logic
+       pass
+   ```
+
+4. Write your `dlt` pipeline:
+   ```py
+   # Modal Secrets are loaded as environment variables which are used here to create the SQLALchemy connection string
+   pg_url = f'postgresql://{os.environ["PGUSER"]}:{os.environ["PGPASSWORD"]}@localhost:{os.environ["PGPORT"]}/{os.environ["PGDATABASE"]}'
+   snowflake_url = f'snowflake://{os.environ["SNOWFLAKE_USER"]}:{os.environ["SNOWFLAKE_PASSWORD"]}@{os.environ["SNOWFLAKE_ACCOUNT"]}/{os.environ["SNOWFLAKE_DATABASE"]}'
+   # Create a pipeline
+   schema = "POSTGRES_DLT_DEV" if dev else "POSTGRES_DLT"
+   pipeline = dlt.pipeline(
+       pipeline_name="task",
+       destination=dlt.destinations.snowflake(snowflake_url),
+       dataset_name=schema,
+       progress="log",
+   )
+   credentials = ConnectionStringCredentials(pg_url)
+   # defines the postgres table to sync (in this case, the "task" table)
+   source_1 = sql_database(credentials, backend="connectorx").with_resources("task")
+   # defines which column to reference for incremental loading (i.e. only load newer rows)
+   source_1.task.apply_hints(
+       incremental=dlt.sources.incremental(
+           "enqueued_at",
+           initial_value=pendulum.datetime(2024, 7, 24, 0, 0, 0, tz="UTC"),
+       )
+   )
+
+    # if there are duplicates, merge the latest values
+   info = pipeline.run(source_1, write_disposition="merge")
+   print(info)
+   ```
+
+## Advanced configuration
+### Modal Proxy
+
+If your database is in a private VPN, you can use [Modal Proxy](https://modal.com/docs/reference/modal.Proxy) as a bastion server (only available to Enterprise customers). We use Modal Proxy to connect to our production read replica by attaching it to the Function definition and changing the hostname to localhost:
+```py
+@app.function(
+    secrets=[
+        modal.Secret.from_name("snowflake-secret"),
+        modal.Secret.from_name("postgres-read-replica-prod"),
+    ],
+    schedule=modal.Cron("24 6 * * *"),
+    proxy=modal.Proxy.from_name("prod-postgres-proxy", environment_name="main"),
+    timeout=3000,
+)
+def task_pipeline(dev: bool = False) -> None:
+    pg_url = f'postgresql://{os.environ["PGUSER"]}:{os.environ["PGPASSWORD"]}@localhost:{os.environ["PGPORT"]}/{os.environ["PGDATABASE"]}'
+```
+
+### Capturing deletes
+
+One limitation of our simple approach above is that it does not capture updates or deletions of data. This isn’t a hard requirement yet for our use cases, but it appears that `dlt` does have a [Postgres CDC replication feature](https://dlthub.com/docs/dlt-ecosystem/verified-sources/pg_replication) that we are considering.
```

Comment:
Please use relative links for the pages in the docs. E.g. `./dlt-ecosystem/...`

Reply:
Thanks @burnash. Updated the link. There's one thing though, the doc is not showing in the GitHub deploy preview here. But when using "npm" locally it shows fine.